prog/dftb+/lib_dftbplus/main.F90

```diff
@@ -304,6 +310,10 @@ subroutine runDftbPlus(env)
 
     call env%globalTimer%stopTimer(globalTimers%postGeoOpt)
 
+    if (allocated(reks)) then
+      call REKS_destroy(reks)
```

Comment:
This is unnecessary. Type `TReksCalc` only contains allocatable arrays and needs no extra finalization. This lines can be removed and also the implementation of the  `REKS_destroy()` routine.